docs/site/src/pages/style/color/Color.js

```diff
@@ -0,0 +1,159 @@
+    color: theme.typography.headline.color,
+  },
+}));
+
+export default function Color(props, context) {
+  const classes = context.styleManager.render(styleSheet);
+  let colorGroups = [];
+  let neutralGroups = [];
+
+  function getColorGroup(styles, color, showAltPalette) {
+    const mainPalette = [50, 100, 200, 300, 400, 500, 600, 700, 800, 900];
+    const altPalette = ['A100', 'A200', 'A400', 'A700'];
+    const cssColor = color.replace(' ', '').replace(color.charAt(0), color.charAt(0).toLowerCase());
+    let colorsList = [];
+    colorsList = mainPalette.map((mainValue) => getColorBlock(styles, cssColor, mainValue));
+
+    if (showAltPalette) {
+      altPalette.forEach((altValue) => {
+        colorsList.push(getColorBlock(styles, cssColor, altValue));
+      });
+    }
+
+    return (
+      <ul className={classes.colorGroup} key={cssColor}>
+        {getColorBlock(styles, cssColor, 500, color)}
+        <div className={classes.blockSpace} />
+        {colorsList}
+      </ul>
+    );
+  }
+
+  function getColorBlock(styles, colorName, colorValue, colorTitle) {
+    const bgColorText = colorName + colorValue;
+    const bgColor = colors[colorName][colorValue];
+    let fgColor = colors.fullBlack;
+    const contrastRatio = getContrastRatio(bgColor, fgColor);
+    let blockTitle;
+    let spaceRequired = false;
+    const colorValueString = colorValue.toString();
+
+    if ((colorValueString.indexOf('A1')) === 0) {
+      spaceRequired = true;
+    }
+
+    if (contrastRatio < 7) fgColor = colors.fullWhite;
+    if (colorTitle) {
+      blockTitle = (
+        <span className={styles.name}>
+          {colorTitle}
+        </span>
+      );
+    }
+
+    const rowStyle = {
+      backgroundColor: bgColor,
+      color: fgColor,
+      listStyle: 'none',
+      padding: 15,
+    };
+
+    if (spaceRequired) {
+      const rowStyleSpace = {
+        ...rowStyle,
+        borderTop: '4px #eeeeee solid',
+      };
+      return (
+        <li style={rowStyleSpace} key={bgColorText}>
+          {blockTitle}
+          {getColorName(styles, bgColorText, bgColor)}
+        </li>
+      );
+    }
+    return (
+      <li style={rowStyle} key={bgColorText}>
+        {blockTitle}
+        {getColorName(styles, bgColorText, bgColor)}
+      </li>
+    );
+  }
+
+  function getColorName(styles, text, colorValue) {
+    return (
+      <div className={styles.colorContainer}>
+        <span>{text}</span>
+        <span className={styles.colorValue}>{colorValue.toUpperCase()}</span>
+      </div>
+    );
+  }
+
+  colorGroups = mainColors.map((maincolor) => getColorGroup(classes, maincolor, true));
+  neutralGroups = neutralColors.map((neutralcolor) => getColorGroup(classes, neutralcolor, false));
+
+  return (
+    <div className={classes.root}>
+      <div>
+        {colorGroups.map((maincolor) => maincolor)}
+      </div>
+      <div>
+        {neutralGroups.map((neutralcolor) => neutralcolor)}
```

Comment:
You are mapping with an identity function. That's pointless: `f o Id = Id o f = f`.